src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -313,23 +313,26 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var triggersArray = new JArray(triggers);
             int count = triggersArray.Count;
 
+            // Form the base minimal result
+            string hostId = await _hostIdProvider.GetHostIdAsync(CancellationToken.None);
```

Comment:
Checking understanding: is the hostId regenerated every time the `GetHostIdAsync` method is called? Double checking the following case: background sync triggers refresh called -> hostId regenerates -> sync triggers hash changes but triggers payload remains the same -> sync triggers payload written to hosting db-> site restarted unnecessarily.